Problems implementing with webpack

Hi, 

I'm currently having some issues trying to inject this into a webpack build. Here's my code showing how I successfully add node modules as a dependency of my angular app. Adding `newrelic-angular` in the same way throws errors (see below). Am I missing something, is this a webpack issue? Has anyone had experience using on a webpack build?

```
angular.module(
  'app',
  [
    require('some-node-module'),
    require('newrelic-angular')
  ]
);
```

Give me the following errors, which i am familiar with due to misspellings etc, which i have checked:

```
Module not found: Error: Cannot resolve module 'newrelic-angular'
uncaught Error: Cannot find module "newrelic-angular"
```

Thanks
G

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "newrelic-angular",
   "description": "New Relic Browser Insights for Angular.js",
-  "main": "lib/module.js",
+  "main": "dist/newrelic-angular.js",
   "version": "0.2.1",
   "dependencies": {
     "angulartics": "^1.0.3",

